spanner/transaction.go

```diff
@@ -1210,6 +1264,10 @@ func NewReadWriteStmtBasedTransactionWithOptions(ctx context.Context, c *Client,
 	t.txReadOnly.txReadEnv = t
 	t.txReadOnly.qo = c.qo
 	t.txOpts = options
+	if c.sc != nil && c.ct == nil {
+		setCommonTags(c)
+	}
```

Comment:
Remove, as this can be replaced by initialization code in `NewClientWithConfig` (see above)